generated/nidmm/__init__.py

```diff
@@ -3,6 +3,6 @@
 
 from nidmm.enums import *          # noqa: F403,F401,H303
 from nidmm.errors import Error     # noqa: F401
-from nidmm.errors import NidmmWarning   # noqa: F401
+from nidmm.errors import NIDMMWarning   # noqa: F401
```

Comment:
I gotta say, I hate the fact that we have a class called `nidmm.NIDMMWarning` and so forth. I wish we could get away with `nidmm.Warning`.

But taking a step back... Having warning class adds a bunch of complexity. Do we even need our own warning classes at all? We never raise these things so clients can't `except` them. What is the advantage to a client in having per-driver Warning classes defined?

Reply:
Yes, I think we want to keep it for filtering.

I like DriverWarning. Changed.